pandana/utils.py

```diff
@@ -0,0 +1,25 @@
+import pandas as pd
+
+
+def reindex(series1, series2):
+    """
+    Reindex the first series by the second series.
+
+    Parameters
+    ----------
+    series1 : pandas.Series
+        Pandas series to reindex
+    series2 : pandas.Series
+        Pandas series to set the index of series1 by
+
```

Comment:
Totally optional - it might be helpful to add a brief note that this is identical to the reindex function in urbansim.